### ISSUE TITLE
no focus style for reblog button

### ISSUE THREAD

**Issue by nightpool** [mastodon/mastodon#4508]
There is no :focus style for the reblog button, although there is a (very faint) focus style for the other fontawesome icons. this makes it very weird to tab through the action bar

see attached gif

![focus-reblog-broken](https://user-images.githubusercontent.com/233815/28933794-7b462f5c-784c-11e7-912d-eb5101990316.gif)

**Comment by nightpool** [comment]
styles are *slightly* more noticeable on the default background, but not by much.
